dt-contacts/user-module.php

```diff
@@ -98,7 +98,7 @@ public static function dt_record_admin_actions( $post_type, $post_id ){
                             <label for="user-email">
                                 <?php esc_html_e( "Email", "disciple_tools" ); ?>
                             </label>
-                            <input name="user-email" id="user-email" type="email" placeholder="user@example.com" required aria-describedby="email-help-text">
+                            <input name="user-email" id="user-email" type="email" value="<?php ( isset( $contact['contact_email'][0]['value'] ) ) ? esc_html_e( $contact['contact_email'][0]['value'], 'disciple_tools' ) : ''; ?>" placeholder="user@example.com" required aria-describedby="email-help-text">
```

Comment:
Hey, `esc_html_e()` add the translation element which isn't needed for the email address value.
`echo esc_html()` would work fine